lib/src/container/store.rs

```diff
@@ -413,6 +449,18 @@ impl ImageImporter {
         })
     }
 
+    ///Given a target diffid, return its corresponding blob
+    //Not fetch all the blobs
+    //How do we use diffid here?
+    pub async fn blob_from_diffid(
+        self,
```

Comment:
I think the problem is here, you want this to be `&self` to borrow instead of move.